packages/common/src/paths.tsx

```diff
@@ -0,0 +1,25 @@
+export const PATH_HOME = '/';
+
+export const PATH_MEDIA = '/m/:id/*';
+export const PATH_PLAYLIST = '/p/:id/*';
+export const PATH_LEGACY_SERIES = '/s/:id/*';
+
+export const PATH_SEARCH = '/q/*';
+
+export const PATH_USER = '/u/*';
+export const PATH_USER_ACCOUNT = '/u/my-account';
+export const PATH_USER_MY_PROFILE = '/u/my-profile/:id';
+export const PATH_USER_FAVORITES = '/u/favorites';
+export const PATH_USER_PAYMENTS = '/u/payments';
+export const PATH_USER_PROFILES = '/u/profiles';
+export const PATH_USER_PROFILES_CREATE = '/u/profiles/create';
+export const PATH_USER_PROFILES_EDIT = '/u/profiles/edit';
+export const PATH_USER_PROFILES_EDIT_PROFILE = '/u/profiles/edit/:id';
+
+export const PATH_ABOUT = '/o/about';
+
+// Returns the path without parent prefix, to be used as nested route
+// I.E.: getNestedPath('/u', '/u/my-account') => '/my-account'
+export const getNestedPath = (parentPath: string, fullPath: string) => {
```

Comment:
The nested paths don't have a leading `/`. Does this change the behavior of the nested routing?

Reply:
No, they were without the slash already and will remain like that. I will update the code comment example.

Reply:
I made it simpler actually. Instead of exporting this util, I use it here and export separate consts. That should keep the code more predictable.